dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -198,11 +182,11 @@ public int getMinutesUntilPasswordResetTokenExpires() {
      * by the Settings Service configuration.
      */
     public String getDataverseSiteUrl() {
-        String hostUrl = System.getProperty(SITE_URL);
+        String hostUrl = settingsService.getValueForKey(SettingsServiceBean.Key.SiteUrl);
         if (hostUrl != null && !"".equals(hostUrl)) {
```

Comment:
Same as above

Reply:
With `StringUtils.isNotBlank(hostUrl)` we check (just in case) if property is present.

Reply:
But if the value is blank it will return null and not empty string. I think we can assume that we will operate on default values and no-one with internal server access will try to sabotage the configuration on the server.

Reply:
Ah nvm this method returns empty string.

Reply:
But I'm still thinking, if there is any reason to set this setting to blank?

Reply:
The bottom line is: if we don't then there is no need for this method whatsoever and we can just replace it with `settingsService.getValueForKey(SettingsServiceBean.Key.SiteUrl);`

Reply:
But this method doesn't only return SiteUrl so what do you mean?

Reply:
After discussion with Krzysiek we decided that getting rid of FQDN would be cleaner. You can use this method with implementation as you said `settingsService.getValueForKey(SettingsServiceBean.Key.SiteUrl);`. Or you can get rid of it and use settingsWrapper directly in xhtml.

Reply:
I've decided to keep `systemConfig.getDataverseSiteUrl()` for mostly aesthetic reasons. Code looks cleaner. 
IMO If we want to change all (many) calls to `systemConfig.getDataverseSiteUrl()` to `settingsService.getValueForKey(SettingsServiceBean.Key.SiteUrl)` we would introduce some unnecessary redundancy.